sdk/core/Azure.Core.TestFramework/src/TestEnvironment.cs

```diff
@@ -168,7 +168,11 @@ public virtual TokenCredential Credential
                     _credential = new ClientSecretCredential(
                         GetVariable("TENANT_ID"),
                         GetVariable("CLIENT_ID"),
-                        GetVariable("CLIENT_SECRET")
+                        GetVariable("CLIENT_SECRET"),
```

Comment:
@pakrym, could you have a look at this change and let me know how you feel about it?

Reply:
It's good if you get `AZURE_AUTHORITY_HOST` for all runs.